packages/datadog-instrumentations/src/mocha.js

```diff
@@ -0,0 +1,91 @@
+const { addHook, channel, AsyncResource } = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const runTestStartCh = channel('ci:mocha:run-test:start')
+const errorCh = channel('ci:mocha:run-test:error')
+const skipCh = channel('ci:mocha:run-test:skip')
+const runTestEndCh = channel('ci:mocha:run-test:end')
+const runTestEndAsyncCh = channel('ci:mocha:run-test:async-end')
+const runTestsEndCh = channel('ci:mocha:run-tests:end')
```

Comment:
What does `run-tests` mean? Is that the test suite? If it is, it would be easier to understand if named `ci:mocha:suite:end`.